### PR TITLE
Camera photos

### LINKED ISSUE #19: Camera Photos
Include camera library to allow users to change their profile pictures.

### PR DESCRIPTION
**Pull request by newtonneto**

### CODE REVIEW

File: src/utils/handleGalleryPermission.ts [hunk 1]
```diff
@@ -24,26 +25,26 @@ const blockedPermission = () => {
...
 
-const handleGalleryPermissions = async (): Promise<PermissionStatus> => {
+const getPermissions = async (type: string): Promise<PermissionStatus> => {
   let result: PermissionStatus;
 
   Platform.OS === 'android'
-    ? (result = await check(PERMISSIONS.ANDROID.READ_EXTERNAL_STORAGE))
-    : (result = await check(PERMISSIONS.IOS.PHOTO_LIBRARY));
+    ? (result = await check(permissionsAndroid[type]))
+    : (result = await check(permissionsIos[type]));
```
Comment (breno-alves) [inline src/utils/handleGalleryPermission.ts]:
```
result = Platform.OS === 'android'
    ? (await check(permissionsAndroid[type]))
    : (await check(permissionsIos[type]));
 ```

**Review by breno-alves — CHANGES_REQUESTED** [review]

File: src/utils/handleGalleryPermission.ts (hunk 1 above)
Comment (newtonneto) [reply src/utils/handleGalleryPermission.ts]: Done

File: src/screens/ProfileDetails/index.tsx [hunk 2]
```diff
@@ -202,16 +206,32 @@ const ProfileDetails = () => {
...
+        });
+      }
+    }
+  };
+
+  const handleCamera = async () => {
+    const permissionStatus = await getPermissions('camera');
+
+    if (permissionStatus === 'granted') {
+      console.log('test');
```
Comment (breno-alves) [inline src/screens/ProfileDetails/index.tsx]: console.log

File: src/utils/getPermissions.ts [hunk 3]
```diff
@@ -24,26 +25,27 @@ const blockedPermission = () => {
...
 };
 
-const handleGalleryPermissions = async (): Promise<PermissionStatus> => {
+const getPermissions = async (type: string): Promise<PermissionStatus> => {
   let result: PermissionStatus;
 
-  Platform.OS === 'android'
-    ? (result = await check(PERMISSIONS.ANDROID.READ_EXTERNAL_STORAGE))
-    : (result = await check(PERMISSIONS.IOS.PHOTO_LIBRARY));
+  result =
```
Comment (breno-alves) [inline src/utils/getPermissions.ts]: use `const` instead of let in this case.

if type is necessary just use cast on it

**Review by breno-alves — CHANGES_REQUESTED** [review]

File: src/screens/ProfileDetails/index.tsx (hunk 2 above)
Comment (newtonneto) [reply src/screens/ProfileDetails/index.tsx]: Done

File: src/utils/getPermissions.ts (hunk 3 above)
Comment (newtonneto) [reply src/utils/getPermissions.ts]: Done

**Review by breno-alves — APPROVED** [review]

**Pull request merged**